lib/unit-ruby/util/api_resource.rb

```diff
@@ -96,7 +97,7 @@ def self.belongs_to(resource_name, class_name: nil)
 
       define_method("#{resource_name}=") do |resource|
         relationships[resource_name] = {
-          data: { type: resource_name, id: resource.id }
+          data: { type: resource_type || resource_name, id: resource.id }
```

Comment:
Shouldn't this be just `resource_type` since it's always a non-nil value from line 88?

Reply:
yess 100%, good catch 😬